rts/Sim/Features/Feature.cpp

```diff
@@ -270,25 +270,20 @@ bool CFeature::AddBuildPower(CUnit* builder, float amount)
 		if (reclaimLeft <= 0.0f)
 			return false;
 
-		const CTeam* builderTeam = teamHandler.Team(builder->team);
+		const auto builderTeam = teamHandler.Team(builder->team);
```

Comment:
Best to put `auto*` instead of `auto`

Reply:
Sure, though keep in mind the `const` applies to the pointer so it will be `auto *const` (and not `const auto*`).